src/plugin.scss

```diff
@@ -3,8 +3,12 @@ Styles for ad integrations.
 */
 
 // Prevent seeking during ad playback
-.vjs-ad-playing.vjs-ad-playing .vjs-progress-control {
-  pointer-events: none;
+.vjs-ad-playing.vjs-ad-playing .vjs-progress-control::after {
```

Comment:
Does sass transform the double colon (`::`) into a single colon (`:`)? IE8 only supports a single colon.